widgets/common/src/filters/RuleValueInput.tsx

```diff
@@ -0,0 +1,63 @@
+import type { FunctionComponent } from 'react';
+import { useEffect, useRef } from 'react';
+
+import { LabelsFilterRuleOperators, FilterRuleRowType, FilterRuleOperator } from './types';
+import LabelKeyDropdown from '../labels/KeyDropdown';
+import LabelValueDropdown from '../labels/ValueDropdown';
+
+interface RuleOperatorDropdownProps {
+    onChange: (value: string[]) => void;
+    operator: FilterRuleOperator;
+    ruleType: FilterRuleRowType;
+    toolbox: Stage.Types.Toolbox;
+    values: string[];
+}
+
+const RuleValueInput: FunctionComponent<RuleOperatorDropdownProps> = ({
+    onChange,
+    operator,
+    ruleType,
+    toolbox,
+    values
+}) => {
+    const { Input } = Stage.Basic;
+    const keyDropdownRef = useRef();
```

Comment:
Optional:

We could specify the type of the `current` property:

```suggestion
    const keyDropdownRef = useRef<HTMLElement>();
```

Reply:
Right, will add it.